R/utils.R

```diff
@@ -76,7 +76,7 @@ file_url <- function(filename) {
   anchor_suffix <- sub(file, "", filename)
```

Comment:
See my main comment, but I believe this will solve your issue in **rgl**

```suggestion
  anchor_suffix <- sub(file, "", filename, fixed = TRUE)
```